src/switchroot/ostree-prepare-root.c

```diff
@@ -397,6 +413,12 @@ main (int argc, char *argv[])
       if (mount (deploy_path, TMP_SYSROOT, NULL, MS_BIND | MS_SILENT, NULL) < 0)
         err (EXIT_FAILURE, "failed to make initial bind mount %s", deploy_path);
     }
+  else
+    {
+#ifdef USE_LIBSYSTEMD
+      sd_journal_send ("MESSAGE=Mounted composefs (signed: %d)", (int)using_composefs_signature, NULL);
```

Comment:
This is not quite right. using_composefs_signature is only set if we applied the signature this boot, which only happens on the first boot of this deployment.

Reply:
Maybe just drop that part, i'm looking at reworking the signature handling.